test/tests/functional/pbs_multi_sched.py

```diff
@@ -647,6 +647,268 @@ def test_multi_fairshare(self):
+        self.server.manager(MGR_CMD_SET, SERVER,
+                            {'job_history_enable': 'True'}, expect=True)
+        # Add entry to the resource group of default scheduler
+        self.scheds['default'].add_to_resource_group('grp1', 100, 'root', 60)
+        self.scheds['default'].add_to_resource_group('grp2', 200, 'root', 40)
+        self.scheds['default'].add_to_resource_group('pbsuser1',
+                                                     101, 'grp1', 40)
+        self.scheds['default'].add_to_resource_group('pbsuser2',
+                                                     102, 'grp1', 20)
+        self.scheds['default'].add_to_resource_group('pbsuser3',
+                                                     201, 'grp2', 30)
+        self.scheds['default'].add_to_resource_group('pbsuser4',
+                                                     202, 'grp2', 10)
+        # Add entry to the resource group of multisched 'sc1'
+        self.scheds['sc1'].add_to_resource_group('grp1', 100, 'root', 60)
+        self.scheds['sc1'].add_to_resource_group('grp2', 200, 'root', 40)
+        self.scheds['sc1'].add_to_resource_group('pbsuser1',
+                                                 101, 'grp1', 40)
+        self.scheds['sc1'].add_to_resource_group('pbsuser2',
+                                                 102, 'grp1', 20)
+        self.scheds['sc1'].add_to_resource_group('pbsuser3',
+                                                 201, 'grp2', 30)
+        self.scheds['sc1'].add_to_resource_group('pbsuser4',
+                                                 202, 'grp2', 10)
+        # Add entry to the resource group of multisched 'sc2'
+        #(sched_priv_sc2 at non-default location)
+        self.scheds['sc2'].add_to_resource_group('grp1', 100, 'root', 60)
+        self.scheds['sc2'].add_to_resource_group('grp2', 200, 'root', 40)
+        self.scheds['sc2'].add_to_resource_group('pbsuser1',
+                                                 101, 'grp1', 40)
+        self.scheds['sc2'].add_to_resource_group('pbsuser2',
+                                                 102, 'grp1', 20)
+        self.scheds['sc2'].add_to_resource_group('pbsuser3',
+                                                 201, 'grp2', 30)
+        self.scheds['sc2'].add_to_resource_group('pbsuser4',
+                                                 202, 'grp2', 10)
+        # Set scheduler iteration
+        sc_iter = {'scheduler_iteration': 7}
+        self.server.manager(MGR_CMD_SET, SCHED, sc_iter, id='default')
+        self.server.manager(MGR_CMD_SET, SCHED, sc_iter, id='sc1')
+        self.server.manager(MGR_CMD_SET, SCHED, sc_iter, id='sc2')
+        # Update scheduler config file
+        formula = 'ceil(fabs(-ncpus*(foo/100.00)*sqrt(100)))'
+        sc_config = {'fair_share': 'True', 'fairshare_decay_time': '24:00:00',
+                     'fairshare_decay_factor': 0.5,
+                     'fairshare_usage_res': formula}
```

Comment:
fairshare_decay_time and fairshare_decay_factor default to those values.  You don't have to set them if you don't want to.